kubernetes-kit-starter/src/main/java/com/vaadin/kubernetes/starter/sessiontracker/SessionSerializationCallback.java

```diff
@@ -0,0 +1,20 @@
+package com.vaadin.kubernetes.starter.sessiontracker;
+
+/**
+ * Error callbacks that are called after a serialization or deserialization
+ * error happens.
+ */
+public interface SessionSerializationCallback {
```

Comment:
nit: this interface targets only errors. Perhaps the class name should also reflect the behavior,  e.g. `SessionSerializationErrorCallback`

Reply:
Or maybe expand it to notify also successful completion?

Reply:
Added callbacks on successful completion.